### DISCUSSION TITLE
Why isn't doctype included by default?

### DISCUSSION THREAD

**Discussion by bryanjamesmiller** [GrapesJS/grapesjs#4271] [Q&A] [answered]
I know there are several issues opened regarding this, and that @artf has provided a workaround (which unfortunately doesn't work very well for us at least in the case where the editor is empty, see https://jsfiddle.net/gratiafide/mgfetw95/6/ - note you cannot drag and drop a new component into the editor if the editor doesn't already have any components in it).   Why isn't `<!DOCTYPE html>` included in the Grapes editor iframe by default?  Why should the extremely outdated "quirks mode" be the default?  Is there a reason for this?

Because of the above issue, we cannot use the doctype workaround provided by @artf and without `<!DOCTYPE html>` some of our blocks render incorrectly (I know others have said other libraries such as tinyMce don't work correctly either).  You will see, for example, the block on the right in the below screenshots have different lengths in the Grapes editor depending on if we have included `<!DOCTYPE html>` or not:
![Screenshot at Apr 19 14-35-02](https://user-images.githubusercontent.com/8781182/164082157-d88fa83f-9331-4297-bfaa-64132daea62e.png)
![Screenshot at Apr 19 14-34-49](https://user-images.githubusercontent.com/8781182/164082164-8fc06615-76c3-4881-bafd-74e7cd573dd8.png)

**Comment by artf** [comment, marked as answer]
Hey I think the first reason was probably because of that bug but I'd ask you to check if this solves your issues in standard mode
```js
grapesjs.init({
 // ...
 canvasCss: '[data-gjs-type="wrapper"] { min-height: 100vh }',
})
```

**Reply by bryanjamesmiller** [reply]
Thank you @artf this appears to be working for us.  I appreciate it very much!
